src/internal.c

```diff
@@ -19825,10 +19843,15 @@ int SendCertificateVerify(WOLFSSL* ssl)
                 args->sigAlgo = ed25519_sa_algo;
 
             if (IsAtLeastTLSv1_2(ssl)) {
-                EncodeSigAlg(ssl->suites->hashAlgo, args->sigAlgo,
-                             args->verify);
+                #if defined(HAVE_ECC) || ( !defined(NO_DH) && !defined(NO_RSA) )
```

Comment:
Please adjust defines around EncodeSigAlg() and SetDigest() and and continue using existing calls.